src/installation/guides/chroot.md

```diff
@@ -3,7 +3,7 @@
 This guide details the process of manually installing Void via a chroot on an
 x86, x86_64 or aarch64 architecture. It is assumed that you have a familiarity
 with Linux, but not necessarily with installing a Linux system via a chroot.
-This guide can be used to create a a "typical" setup, using a single partition
+This guide can be used to create a "typical" setup, using a single partition
```

Comment:
Please run `vmdfmt` in the file (or just move the `on` from the line below to this line ;)